beef_tx.go

```diff
@@ -124,3 +124,54 @@ func getParentTransactionsForInput(ctx context.Context, client ClientInterface,
 
 	return nil, fmt.Errorf("transaction is not mined yet (tx.ID: %s)", inputTx.ID) // TODO: handle it in next iterration
 }
+
```

Comment:
1. It's not beef resposibility to save some transactions. Move it to `paymail_service_provider` (or maybe to `record_tx_strategy_external_incoming` ? I'm not sure if it's possible)
2. Do not sync any transaction here. Let the task manager do its job.
3. Instead of `fmt.Printl()` use `DebugLog()`
4. Use new `GetTransactionsByIDs()` to get all transactions in one roundtrip 
5. Do not save Tx and SyncTx in to separate calls! Create SyncTx object and add it as child model to Transaction :
```
sync.transaction = tx
	tx.syncTransaction = sync

Reply:
Done